### ISSUE TITLE
After h1 "about" all other headings should be h2

### ISSUE THREAD

**Issue by femmebot** [18F/nsf-sbir#695]
## Expected Behavior
- [x] View the page outline. Ensure the headings are logical. One h1 per page, sub headings don't skip a level, elements of similar import are on the same heading level

see: 508 compliance report [About N° 90](https://docs.google.com/spreadsheets/d/1yWVeVHPJtoqsCTEwnYalqDTSXNkr5E_Bn20msVmkP-8/edit#gid=0)
